test/e2e/network/network_policy.go

```diff
@@ -906,6 +951,94 @@ var _ = SIGDescribe("NetworkPolicy [LinuxOnly]", func() {
 			testCannotConnect(f, f.Namespace, "client-a", service, allowedPort)
 		})
 
+		ginkgo.It("should work with Ingress,Egress specified together [Feature:NetworkPolicy]", func() {
+			const allowedPort = 80
+			const notAllowedPort = 81
+			protocolUDP := v1.ProtocolUDP
+
+			nsBName := f.BaseName + "-b"
```

Comment:
Wouldn't this collide with the other test case (when running in parallel) if we append the same suffix (the base namespace is the same)?

Reply:
it wouldn’t, namespaces has unique prefix

Reply:
Example from log
```
STEP: Creating a server pod server in namespace network-policy-2306
STEP: Creating a server pod pod-b in namespace network-policy-b-6489
```